student-info-system/sis/models.py

```diff
@@ -222,3 +262,24 @@ def name(self):
 
     def __str__(self):
         return self.name()
+
+
+""" making it so users know about roles, but without overhead of subclassing """
```

Comment:
Is this supposed to be documentation? If not we  should use # for internal comments

Reply:
done